src/engine/source/kvdb2/include/kvdb2/iKVDBHandler.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef _IKVDBHANDLER_H
```

Comment:
I think it should be `_I_KVDB_HANDLER_H`

Reply:
Will be fixed in later documentation step.